backend/apps/slack/common/handlers/contribute.py

```diff
@@ -0,0 +1,94 @@
+"""Handler for OWASP Contribute Slack functionality."""
+
+from __future__ import annotations
+
+from django.utils.text import Truncator
+
+from apps.common.constants import NL
+from apps.slack.blocks import get_pagination_buttons, markdown
+from apps.slack.common.constants import TRUNCATION_INDICATOR
+from apps.slack.common.presentation import EntityPresentation
+from apps.slack.constants import FEEDBACK_CHANNEL_MESSAGE
+from apps.slack.utils import escape
+
+
+def get_blocks(
+    page=1, search_query: str = "", limit: int = 10, presentation: EntityPresentation | None = None
+):
+    """Get contribute blocks."""
+    from apps.owasp.api.search.issue import get_issues
+    from apps.owasp.models.contribute import Contribute
```

Comment:
plz use` issue` model from ` apps/github/models` don't create extra model , it is a duplicate 
```suggestion
    from apps.github.models.issue import Issue
```